### ISSUE TITLE
Allow the creation of new arbitrary send to's

### ISSUE THREAD

**Issue by ezhes** [shusain93/Andromeda-iMessage#2]

**Comment by cawilliamson** [comment]
Hey - firstly, this combo of apps is AWESOME! Great work!

Let me know a PayPal address and I'd be happy to donate to the project! :)

As a side note though - the title of this issue is a bit confusing. Is the issue here that you can't start a *new* conversation using the Android app or am I missing something?

Really what I'm asking here is... is it possible to create a *new* conversation currently and if so, how? :)

Thanks again - awesome work!

**Comment by ezhes** [comment]
Yes the issue is as you say. It's an open issue still, haven't gotten around to it yet sorry.
